ClassLibrary/Employee.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace ClassLibrary
+{
+    public class Employee
+    {
+        private string firstName;
```

Comment:
name private fields according to common C# style : add undescore before name, like _firstName

Reply:
done